Tween/Easings/EasingFunctionStore.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+
+namespace DUCK.Tween.Easings
+{
+	public class EasingFunctionStore
+	{
+		public static Dictionary<string, Func<float, float>> EasingFunctions { get; set; }
```

Comment:
Does this dictionary need to have a public setter ?

Reply:
Yeah it should not